### ISSUE TITLE
install xcat on sles11 didn't work

### ISSUE THREAD

**Issue by penguhyang** [xcat2/xcat-core#285]
```
c910f02c09p12:~/xcat2 # more /etc/SuSE-release
SUSE Linux Enterprise Server 11 (ppc64)
VERSION = 11
PATCHLEVEL = 0
c910f02c09p12:~/xcat2 # zypper lr
# | Alias                                | Name                                 | Enabled | Refresh
--+--------------------------------------+--------------------------------------+---------+--------
1 | 1                                    | SuSE-Linux-pkg0                      | Yes     | Yes
2 | SUSE-Linux-Enterprise-Server-11 11-0 | SUSE-Linux-Enterprise-Server-11 11-0 | Yes     | Yes
c910f02c09p12:~/xcat2 # zypper ar file:///root/xcat2/xcat-dep/sles11/ppc64 xCAT-dep
Adding repository 'xCAT-dep' [done]
Repository 'xCAT-dep' successfully added
Enabled: Yes
Autorefresh: No
URI: file:/root/xcat2/xcat-dep/sles11/ppc64

c910f02c09p12:~/xcat2 # zypper ar file:///root/xcat2/xcat-core xcat-core
Adding repository 'xcat-core' [done]
Repository 'xcat-core' successfully added
Enabled: Yes
Autorefresh: No
URI: file:/root/xcat2/xcat-core

c910f02c09p12:~/xcat2 # zypper ref
Repository 'SuSE-Linux-pkg0' is up to date.
Repository 'SUSE-Linux-Enterprise-Server-11 11-0' is up to date.
Retrieving repository 'xCAT-dep' metadata [done]
Building repository 'xCAT-dep' cache [done]
Retrieving repository 'xcat-core' metadata [done]
Building repository 'xcat-core' cache [done]
All repositories have been refreshed.
c910f02c09p12:~/xcat2 # zypper lr
# | Alias                                | Name                                 | Enabled | Refresh
--+--------------------------------------+--------------------------------------+---------+--------
1 | 1                                    | SuSE-Linux-pkg0                      | Yes     | Yes
2 | SUSE-Linux-Enterprise-Server-11 11-0 | SUSE-Linux-Enterprise-Server-11 11-0 | Yes     | Yes
3 | xCAT-dep                             | xCAT-dep                             | Yes     | No
4 | xcat-core                            | xcat-core                            | Yes     | No
c910f02c09p12:~/xcat2 # zypper install xCAT
Loading repository data...
Reading installed packages...
'xCAT' not found.
Resolving package dependencies...
Nothing to do.
```

**Comment by whowutwut** [comment]
@penguhyang , The formatting of the text in the description is all funny because the characters are being interpreted as markdown symbols.  Take a look at this: 
https://guides.github.com/features/mastering-markdown/#examples

You can surround the console text with back ticks (```) to format it into code, it will help make it a lot easier to read the issue.

**Comment by penguhyang** [comment]
Sorry about that, it looks better now.

**Comment by whowutwut** [comment]
@penguhyang  No worries at all! Thanks for making the changes.

**Comment by penguhyang** [comment]
Since sles11.0 is not supported by xCAT, this issue will make no sense.

**Comment by whowutwut** [comment]
are we dropping SLES 11 support in xCAT?

**Comment by daniceexi** [comment]
@whowutwut 
In the plan sles11.0 should be supported, but since the support of sles11.0 was 4 or more years ago. some changes in the xCAT rpm package does not work for sles11.0.

Two issues: 
1. the metadata of xcat-core repo does not work for sles11.0 that zypper cannot find any pkgs in the xcat-core repo.
2.  xcat-dep missed a perl-Sys-Virt for sles11.0. This rpm has been included in sles11.3/sles11.4, so we did not include it in xcat-dep.

My thought was we did not see customer was using sles11.0, so we can take this risk that make it as a known issue and don't fix it until certain customer brings it up.
